yb-voyager/src/dbzm/dbzm.go

```diff
@@ -9,35 +9,57 @@ import (
 	"time"
 
 	"github.com/tebeka/atexit"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
 	"gopkg.in/natefinch/lumberjack.v2"
 
 	log "github.com/sirupsen/logrus"
 )
 
 var DEBEZIUM_DIST_DIR, DEBEZIUM_CONF_FILEPATH string
 
+const DEBEZIUM_BREW_VERSION = "2.2.0-voyager-1.3"
+const DEBEZIUM_VERSION = "2.2.0"
+
 type Debezium struct {
 	*Config
 	cmd  *exec.Cmd
 	err  error
 	done bool
 }
 
-func init() {
+func findDebeziumDistribution() error {
 	if distDir := os.Getenv("DEBEZIUM_DIST_DIR"); distDir != "" {
 		DEBEZIUM_DIST_DIR = distDir
 	} else {
-		DEBEZIUM_DIST_DIR = "/opt/yb-voyager/debezium-server"
+		possiblePaths := []string{
+			"/opt/homebrew/Cellar/debezium@" + DEBEZIUM_VERSION + "/" + DEBEZIUM_BREW_VERSION + "/debezium-server",
+			"/usr/localCellar/debezium@" + DEBEZIUM_VERSION + "/" + DEBEZIUM_BREW_VERSION + "/debezium-server",
+			"/opt/yb-voyager/debezium-server"}
+		for _, path := range possiblePaths {
+			if utils.FileOrFolderExists(path) {
+				DEBEZIUM_DIST_DIR = path
+				break
+			}
+		}
+		if DEBEZIUM_DIST_DIR == "" {
+			err := fmt.Errorf("Debezium dist directory not found")
```

Comment:
Have a proper error message that tells the problem and possible way to solve the problem.
```
fmt.Errorf("Could not find debezium-server directory in any of %v. Either install debezium-server or provide its path in the DEBEZIUM_DIST_DIR env variable", possiblePaths)
```